cmake/BTApplication.cmake

```diff
@@ -28,6 +28,11 @@ ENDIF()
 
 FIND_PACKAGE(Sword 1.8.1 REQUIRED)
 
+# Translation related
+FIND_PACKAGE(PkgConfig REQUIRED)
+FIND_PACKAGE(Gettext REQUIRED)
```

Comment:
`FIND_PACKAGE(Gettext REQUIRED)` is unused and only searches for the command-line tools. Instead add `find_package(Intl REQUIRED)` and add `Intl::Intl` to the `target_link_libraries(bibletime_backend PRIVATE)` call to use `::gettext()`.